bedrock/settings/base.py

```diff
@@ -182,6 +182,19 @@ def get_dev_languages():
 }
 
 
+def lazy_lang_group():
+    from django.conf import settings
+
+    groups = {}
+    langs = settings.DEV_LANGUAGES if settings.DEV else settings.PROD_LANGUAGES
+    for lang in langs:
+        if '-' in lang:
```

Comment:
Surely an edge case, but what would happen here for `ja` and `ja-JP-mac`? Seems `ja` would never be derived, as sending `ja` in would expand only to `ja-JP-mac`, right?

I can't see this being a practical issue any time soon, but could be a weird bug somewhere down the line.

Typing out loud here, but what would you think about removing groups of one, e.g. `ja-JP-mac`, `fy-NL` and `ga-IE`?